### PR TITLE
WIFI-2680: Revert inactive fw version in case of failed upgrade

### PR DESCRIPTION
**Pull request by shimmy568**
Signed-off-by: Owen Anderson <owenthomasanderson@gmail.com>

### CODE REVIEW

File: feeds/wlan-ap/opensync/files/bin/flash-firmware [hunk 1]
```diff
@@ -34,11 +34,13 @@ fi
 
 # Set the current version as inactive before the upgrade
 FW_VERSION=`sed -n 's/FW_IMAGE_ACTIVE:\(.*\)/\1/p' < /usr/opensync/.versions`
+FW_BACKUP=$(uci get system.tip.inactivefw)
 uci set system.tip.inactivefw="${FW_VERSION}"
 uci commit
 
 /sbin/sysupgrade $IMGFILE
 if [ "$?" != "0" ] ; then
+  uci set system.tip.inactivefw="${FW_BACKUP}"
```
Comment (yashvardhan-netex) [inline feeds/wlan-ap/opensync/files/bin/flash-firmware:43]: Missing uci commit system

File: feeds/wlan-ap/opensync/files/bin/flash-firmware [hunk 2]
```diff
@@ -34,11 +34,14 @@ fi
...
+  uci commit
```
Comment (NagConnectus) [inline feeds/wlan-ap/opensync/files/bin/flash-firmware:44]: you can do **uci commit system**. Since we are trying to commit system configuration file.

**Review by NagConnectus — APPROVED** [review]
looks good to me!

**Review by yashvardhan-netex — APPROVED** [review]
Looks good!

**Review by RickSommerville — APPROVED** [review]

**Comment by shimmy568** [comment]
Sorry about all those reopen/closes. Github was doing some weird stuff because of all the force pushes.

**Comment by RickSommerville** [comment]
Merged 78b1f77a, thanks

**Pull request closed without merging**